src/BrowscapSiteConsole.php

```diff
@@ -12,13 +14,18 @@ public function __construct()
     {
         parent::__construct('Browscap Website', 'dev-master');
 
-        // @todo Config should come from a shared container, this is terrible
-        $config = require __DIR__ . '/../config/config.php';
+        /** @var ContainerInterface $container */
+        $container = require __DIR__ . '/../config/container.php';
+
+        $dbConfig = $container->get('Config')['db'];
 
         $commands = [
             new Command\GenerateStatisticsCommand(new AnalyseStatistics(
-                new \PDO($config['db']['dsn'], $config['db']['user'], $config['db']['pass'])
+                new \PDO($dbConfig['dsn'], $dbConfig['user'], $dbConfig['pass'])
```

Comment:
May you move `new AnalyseStatistics(...)` to the container config and write here `$container->get(AnalyseStatistics::class)?